src/memray/reporters/flamegraph.py

```diff
@@ -83,22 +211,35 @@ def _from_any_snapshot(
         memory_records: Iterable[MemorySnapshot],
         native_traces: bool,
         temporal: bool,
+        inverted: Optional[bool] = None,
     ) -> "FlameGraphReporter":
-        root: Dict[str, Any] = {
+        # TODO?
+        if temporal and inverted:
+            raise NotImplementedError(
```

Comment:
Have you given any thought to what it would take to extend this to work for temporal mode? It should definitely be a separate PR, this one is long enough - but I don't think it would be too tough to implement.